app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,94 @@
+<div class="ui hidden divider"></div>
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="eight wide column">
+      <h2 class="ui header">
+        {{t 'Discount Codes'}}
+      </h2>
+    </div>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      <button class="ui teal button">
+        <i class="plus icon"></i>
+        {{t 'Create a discount'}}
+      </button>
+    </div>
+  </div>
+  <div class="row">
```

Comment:
Hi @anu-007! Currently, there is no spacing at left. You can add a left margin of 10px which will make everything aligned on screen and looks better! :) 
![image](https://cloud.githubusercontent.com/assets/12701036/26781260/31f3af10-4a0b-11e7-82f7-cf8772181745.png)